packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -1147,22 +1147,18 @@ describe("IModelTransformer", () => {
   });
 
   it("remap root subject to root subject", async () => {
-    const sourceIModelName: string = "source";
-    const targetIModelName: string = "target";
-    const sourceIModelFile: string = path.join(
-      outputDir,
-      `${sourceIModelName}.bim`
+    const sourceIModelName: string = "source.bim";
+    const targetIModelName: string = "target.bim";
+    const sourceIModelFile: string = path.join(outputDir, sourceIModelName);
+    const targetIModelFile: string = path.join(outputDir, targetIModelName);
+    IModelTransformerTestUtils.prepareOutputFile(
```

Comment:
Would it work to get rid of sourceIModelName and targetIModelName?

Then just set sourceIModelFile = to the return of prepareOutputFile?